### ISSUE TITLE
I can't do make

### ISSUE THREAD

**Issue by ValeriaVis** [rahvaalgatus/rahvaalgatus#300]
I try do make and get it
make -C app compile
make[1]: Entering directory '/home/venus/rahvaalgatus/app'
make[2]: Entering directory '/home/venus/rahvaalgatus/app'
Compiling public/assets/html5.js…
Compiling public/assets/inputs.js…
Compiling public/assets/hwcrypto.js…
Compiling public/assets/editor.js…
Compiling public/assets/local.js…

/home/user/rahvaalgatus/tmp/local_governments.geojson.json:2

^
ParseError: Unexpected token
Makefile:54: recipe for target 'public/assets/local.js' failed
make[2]: *** [public/assets/local.js] Error 1
make[2]: Leaving directory '/home/user/rahvaalgatus/app'
Makefile:16: recipe for target 'love' failed
make[1]: *** [love] Error 2
make[1]: Leaving directory '/home/user/rahvaalgatus/app'
Makefile:72: recipe for target 'javascripts' failed
make: *** [javascripts] Error 2
How to fix ?

**Comment by moll** [comment]
Hey,

Are you a friend of @VenusKanami by any chance? :P

Anyways, if you look at the error message, it hints there's something wrong with the `tmp/local_governments.geojson.json` file. What's inside it?

**Comment by ValeriaVis** [comment]
yes. Nothing a create this file and folder

**Comment by moll** [comment]
I'm sorry, could you rephrase that? By the way, @VenusKanami possibly had the same issue as you at https://github.com/rahvaalgatus/rahvaalgatus/issues/298. See if that helps.

I've also ensured the `tmp` directory always gets created when running the Makefile in `assets`. Try that the latest commit, too.

**Comment by ValeriaVis** [comment]
One question please. how i can up this pjojetc for npm or mp2 ?

**Comment by moll** [comment]
I'm sorry, I don't understand the question.

**Comment by ValeriaVis** [comment]
how i can up this project on npm or mp2 ?

**Comment by moll** [comment]
Still don't understand, sorry.

**Comment by ValeriaVis** [comment]
pm2 start index.json
/usr/lib/node_modules/pm2/node_modules/chalk/source/index.js:103
        ...styles,
        ^^^

SyntaxError: Unexpected token ...
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:549:28)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
    at Module.require (module.js:504:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/lib/node_modules/pm2/constants.js:10:14)

**Comment by moll** [comment]
Sorry, I don't know anything about PM2. If you just want to run the app locally, see what `make web` does.

Though I do suspect from the error that PM2 wants a newer Node.js version. I don't know how well Rahvaalgatus will work on new Node.js versions, however.

**Comment by moll** [comment]
I'll close the issue as we solved the original problem, though we can carry on chatting.
